.github/workflows/build-windows-installer.yml

```diff
@@ -317,14 +317,14 @@ jobs:
       fail-fast: false
       matrix:
         os:
-          - name: 2019
-            matrix: 2019
-            runs-on:
-              intel: windows-2019
           - name: 2022
             matrix: 2022
             runs-on:
               intel: windows-2022
+          - name: 2015
+            matrix: 2015
+            runs-on:
+              intel: windows-2015
```

Comment:
I'm going to guess this is supposed to be `windows-2025`

Reply:
[`e8e14ee` (#19670)](https://github.com/Chia-Network/chia-blockchain/pull/19670/commits/e8e14ee44c787d34cf49fc9c870b3875159f718c)